### PR TITLE
Rename function and remove needless thread

### PR DESCRIPTION
**Pull request by ghost**
**Describe Pull request**
Removed needless thread I thought was needed but it was tested and found out not to be the case and renamed the function MenuGarage to OpenMenuGarage to better label its intent

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] (Be honest) **Yes**
- Does your code fit the style guidelines? [yes/no] **Yes**
- Does your PR fit the contribution guidelines? [yes/no] **Yes**

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -143,42 +143,40 @@ local function CreateZone(type, number)
...
-            })
-            if type == 'stores' then
-                CurrentLocation.zoneCombo = boxZones
+        end
+        local function exitZone()
+            if type == 'vehicle' then
+                ShowMarker(false)
+            elseif type == 'stores' then
+                ShowMarker(false)
+                SetDelivering(false)
```
Comment (Manason) [inline client/main.lua]: Can one even have functions within functions like this? If needing to move them out of the OpenMenuGarage() function consider renaming to be more descriptive.

Comment (D4isDAVID) [reply client/main.lua]: Indeed, this is valid Lua. Functions within functions are a thing, as are variables within functions.

Comment (D4isDAVID) [reply client/main.lua]: And no need to move them out since they are only used here.

Comment (Manason) [reply client/main.lua]: Hmm.. Even so I would go out on a limb and say this should never be done as it seems likely to lead to frail code. These should probably be moved as anonymous functions within the box function or extracted out altogether.

Comment (D4isDAVID) [reply client/main.lua]: Agreed, should be anonymous functions

Comment (ghost) [reply client/main.lua]: I'll move them as anonymous functions within the box function, test, then post my changes

**Review by Manason — APPROVED** [review]

**Review by D4isDAVID — APPROVED** [review]
LGTM

**Pull request merged**